src/toil/__init__.py

```diff
@@ -214,6 +216,22 @@ def applianceSelf(forceDockerAppliance=False):
         return checkDockerImageExists(appliance=appliance)
 
 
+def customDockerInitCmd():
+    """
+    Returns the custom command (if any) provided through the ``TOIL_CUSTOM_DOCKER_INIT_COMMAND``
```

Comment:
The new variable needs to be documented in https://github.com/DataBiosphere/toil/blob/master/docs/appendices/environment_vars.rst I think.

Reply:
Done!